network/reset_wifi.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -e
+
+FILECONDITION="/etc/sysconfig/connman"
+DEFAULTPASSWORD="1234567890"
```

Comment:
Please make this a "wrapper script" over the current one to reset network configure